portal/src/graphql/portal/query/appAndSecretConfigQuery.ts

```diff
@@ -0,0 +1,32 @@
+import { gql, QueryResult, useQuery } from "@apollo/client";
+import { client } from "../../portal/apollo";
+import { AppAndSecretConfigQuery } from "./__generated__/AppAndSecretConfigQuery";
+
+export const appAndSecretConfigQuery = gql`
+  query AppAndSecretConfigQuery($id: ID!) {
+    node(id: $id) {
+      __typename
+      ... on App {
+        id
+        effectiveAppConfig
+        rawAppConfig
+        rawSecretConfig
+      }
+    }
+  }
+`;
+
+export const useAppAndSecretConfigQuery = (
```

Comment:
What about we remove the existing query and mutation that only works on config? The newly added query and mutation are more powerful.

Reply:
Then we need to query secret config, and pass it to mutation even if it is not used at all?

Reply:
OK. Let's keep both them